aptos-move/framework/aptos-token-objects/sources/aptos_token.move

```diff
@@ -373,6 +373,7 @@ module aptos_token_objects::aptos_token {
             token::creator(*token) == signer::address_of(creator),
             error::permission_denied(ENOT_CREATOR),
         );
+
```

Comment:
Are there any changes in this module? I saw the doc is updated but no code change here.

Reply:
yeah undo the new line changes here